css/style.css

```diff
@@ -12,3 +12,57 @@ font-family: 'Raleway', sans-serif;
 *{
     font-family: 'Raleway', sans-serif;
 }
+
+
+/* start footer */
+
+footer{
+    padding: 25px 0px;
```

Comment:
I think it needs height: 150px
and if you change it then you have to change the padding to 50px 0px;

Reply:
Thanks for your suggestion, I tried it but found the height is just too big compared to the content. On the other hand, I have taken care that the top and bottom margins are not equal, so thank you again!